repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneRequestHandler.scala

```diff
@@ -146,7 +146,7 @@ class KeystoneRequestHandler(identityServiceUri: String, akkaServiceClient: Akka
 
     val akkaResponse = Try(akkaServiceClient.get(
       s"$TOKEN_KEY_PREFIX$validatableToken",
-      s"$identityServiceUri$TOKEN_ENDPOINT/$validatableToken",
+      s"$identityServiceUri$TOKEN_ENDPOINT/$validatableToken${getApplyRcnRoles(applyRcnRoles)}",
```

Comment:
I find the individual parts easier to read that way. I can actually tell what the string is supposed to be. But that might just be me.